src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -120,61 +120,27 @@ public class MethodologyService : IMethodologyService
                 });
         }
 
-        public async Task<Either<ActionResult, List<TitleAndIdViewModel>>> GetAdoptableMethodologies(Guid publicationId)
+        public async Task<Either<ActionResult, List<MethodologySummaryViewModel>>> GetAdoptableMethodologies(
+            Guid publicationId)
         {
             return await _persistenceHelper
                 .CheckEntityExists<Publication>(publicationId)
                 .OnSuccess(_userService.CheckCanAdoptMethodologyForPublication)
                 .OnSuccess(async publication =>
                 {
                     var methodologies = await _methodologyParentRepository.GetUnrelatedToPublication(publication.Id);
-                    return methodologies
-                        .Select(m => new TitleAndIdViewModel(m.Id, m.OwningPublicationTitle))
-                        .ToList();
+                    var latestVersions = await methodologies.SelectAsync(methodology =>
```

Comment:
Ugh, this reminds me that we should probably add the `System.Linq.Async` package into admin at some point

Reply:
Ah yep, I actually started doing this as I added it to the `Content.Model` package to use it there and then started updating all the dependant projects using `Content.Model` which turned into a much bigger change and abandoned it at that point! I think it makes sense to do it as a separate piece of work.

Reply:
Yep I agree!